app/controllers/policies_controller.rb

```diff
@@ -1,16 +1,16 @@
 class PoliciesController < ApplicationController
   before_action :authenticate_user!
   before_action :new_policy,only: [:new, :user_list, :category_fields]
-  before_action :policies_params, only: [:create, :update]
+  before_action :policies_params, only: [:create, :update, :new_user]
   before_action :set_policy, only: [:edit, :update , :destroy]
   def new
   end
 
   def index
     if (params[:search])
-      @policies = paginated(Policy.search(params[:search]).order(created_at: :asc))
+      @policies = (Policy.search(params[:search]).order(created_at: :asc))
```

Comment:
@KaranBamaniya-BTC : WE need to search the data by ajax & not to reload the form , use remote-ajax for this, 

Create a separate task for this.